packages/synapse-react-client/test/containers/widgets/query-filter/FacetFilterControls.test.tsx

```diff
@@ -161,57 +164,6 @@ describe('FacetFilterControls tests', () => {
   })
 
   describe('handling child component callbacks', () => {
-    it('should propagate enum update correctly', async () => {
```

Comment:
These tested the callbacks passed to a mocked `EnumFacetFilter`. Those callbacks are no longer passed, so I just removed the tests.